### PR TITLE
added pip instructions

### PR DESCRIPTION
**Pull request by aniemerg**
Pip instructions for gateway installation

### CODE REVIEW

File: ethereum-gateway-installation.md [hunk 1]
```diff
@@ -33,34 +35,48 @@ Currently, we are only distributing our docker image from a private Docker Hub r
...
 
 ### Pip Installation
 
 #### (Coming soon)
 
+This section describes the installation of the bloXroute Gateway using pip. 
+
+1. Determine your node ID for your ethereum full node. You can retrieve your node ID using the ethereum rpc command [admin.nodeInfo](https://github.com/ethereum/go-ethereum/wiki/Management-APIs#admin_nodeinfo). 
+
+1. Install via pip
```
Comment (aspin) [inline ethereum-gateway-installation.md]: There's a couple more instructions that necessary, depending on the system. See https://pypi.org/project/bloxroute-gateway-aspin/#description. Maybe link to here as well? (will fix link later)

File: ethereum-gateway-installation.md [hunk 2]
```diff
@@ -33,34 +35,48 @@ Currently, we are only distributing our docker image from a private Docker Hub r
...
    ```
 
 
 ### Pip Installation
 
 #### (Coming soon)
 
+This section describes the installation of the bloXroute Gateway using pip. 
+
+1. Determine your node ID for your ethereum full node. You can retrieve your node ID using the ethereum rpc command [admin.nodeInfo](https://github.com/ethereum/go-ethereum/wiki/Management-APIs#admin_nodeinfo).
```
Comment (sergeyi23) [inline ethereum-gateway-installation.md]: I would add clarification for node id:
The hexadecimal node ID is encoded in the username portion of the enode URL, separated from the host by an @ sign (https://github.com/ethereum/wiki/wiki/enode-url-format).

The problem is that `admin.nodeInfo` returns an object that contains a field called `id` that may be confusing. It is another id.

File: ethereum-gateway-installation.md [hunk 3]
```diff
@@ -33,34 +35,48 @@ Currently, we are only distributing our docker image from a private Docker Hub r
...
 | blockchain-network | Blockchain network gateway is connecting to (Mainnet or Testnet). This test is only on the Mainnet. | Mainnet |
+| node-public-key | node ID of your ethereum node. You can retrieve your node ID using the ethereum rpc command [admin.nodeInfo](https://github.com/ethereum/go-ethereum/wiki/Management-APIs#admin_nodeinfo).  | N/A |
 
 **Optional Gateway Parameters**
 
 These parameters should not be changed unless needed. 
 
 | Parameter | Description | Example |
-| --------- | ----------- | -------------|
+| --------- | ----------- | -------------|\
```
Comment (sergeyi23) [inline ethereum-gateway-installation.md]: I think this breaks layout of the table - `\` in the end of line

File: ethereum-gateway-installation.md (hunk 1 above)
Comment (aniemerg) [reply ethereum-gateway-installation.md]: Added a link.

File: ethereum-gateway-installation.md (hunk 2 above)
Comment (aniemerg) [reply ethereum-gateway-installation.md]: Added that information.

File: ethereum-gateway-installation.md (hunk 3 above)
Comment (aniemerg) [reply ethereum-gateway-installation.md]: Fixed.

File: ethereum-gateway-installation.md [hunk 4]
```diff
@@ -33,15 +35,26 @@ Currently, we are only distributing our docker image from a private Docker Hub r
    docker pull bloxroute/bxgateway:latest
    ```
 	
-5. Run the docker as a daemon. The gateway accepts various parameters as described in the Gateway Parameters section below. The parameters may be passed in the run command to provide the IP address and port of the full node, etc. If using the "external-port" parameter, please update "-p 9001:9001" to "-p (new port):(new port)".
+5. Run the docker as a daemon. The gateway accepts various parameters as described in the Gateway Parameters section below. The parameters may be passed in the run command to provide the IP address and port of the full node, etc. If using the "external-port" parameter, please update "-p 9001:9001" to "-p (new port):(new port)". Be sure to replace "\<YOUR NODE ID\>" with your node id.
```
Comment (aspin) [inline ethereum-gateway-installation.md]: probably rename YOUR NODE ID to `YOUR NODE PUBLIC KEY`

**Pull request merged**